core/src/main/java/com/netflix/conductor/core/reconciliation/DefaultWorkflowSweeper.java

```diff
@@ -33,23 +34,23 @@
 
 @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
 @Component
-@ConditionalOnProperty(name = "conductor.workflow-sweeper.enabled", havingValue = "true", matchIfMissing = true)
-public class WorkflowSweeper {
+@ConditionalOnProperty(name = "conductor.default-workflow-sweeper.enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
A custom poller could be written that uses the `WorkflowSweeper`. I think the `@Conditional` can be removed.

Reply:
removed